py/server/tests/test_arrow.py

```diff
@@ -6,6 +6,7 @@
 from datetime import datetime
 from typing import List, Any
 
+import jpy
```

Comment:
Why the need for import jpy?  I don't see any additions to the code; and if we are requiring it, shouldn't the module we are importing from do that for us?